lib/rust/parser/src/macros.rs

```diff
@@ -0,0 +1,40 @@
+//! The macro system for the Enso parser.
+
+pub mod definition;
+pub mod literal;
+pub mod registry;
+
+use crate::prelude::*;
+use crate::prelude::logger::*;
+
+use crate::macros::definition::Definition;
+use crate::macros::registry::Registry;
+
+
+// ================
+// === Resolver ===
+// ================
+
+/// The Enso macro resolver.
+#[derive(Clone,Debug,PartialEq)]
+pub struct Resolver<Logger> {
+    /// The macro registry.
+    registry : Registry,
+    /// The logger for the macro resolver.
+    logger : Logger
```

Comment:
remove these docs. Doc telling that "logger" is a logger" and "macro::Registry" is a macro registry are useless. Add allow no docs and it iwll be cleaner